src/main/java/com/upversionlab/controller/ProductController.java

```diff
@@ -14,15 +15,16 @@
 @RequestMapping("/products")
 public class ProductController {
     private Map<Integer, Product> products = new HashMap<>();
+    private static final String PRODUCT_RESOURCE_NAME = Product.class.getSimpleName();
```

Comment:
Constants should come first (and be probably be separated by an empty line from other attributes).